src/pretalx/schedule/models/schedule.py

```diff
@@ -72,6 +73,8 @@ def freeze(self, name, user=None, notify_speakers=True):
         with suppress(AttributeError):
             del wip_schedule.event.current_schedule
 
+        export_schedule_html.apply_async(args=(self.event.id,))
```

Comment:
I'd prefer if we'd pass all celery task arguments as `kwargs` always.

Reply:
sure thing, added to TODOs.

Reply:
✅